common/src/chain/transaction/signature/inputsig/classical_multisig/authorize_classical_multisig.rs

```diff
@@ -233,7 +233,7 @@ pub fn sign_classical_multisig_spending(
         return Err(ClassicalMultisigSigningError::SpendeePrivateChallengePublicKeyMismatch);
     }
     let signature = private_key
-        .sign_message(&msg)
+        .sign_message(&msg, randomness::make_true_rng())
```

Comment:
I had to change this and pass rng from outside

Reply:
Sounds good.